stable/airflow/values.yaml

```diff
@@ -1167,6 +1167,12 @@ rbac:
   ##   which is required for the KubernetesPodOperator() to function
   ##
   create: true
+  ## if the created RBAC Role has GET/LIST on event Resources
```

Comment:
I know its a nit-pick, but please add new-line between `create: true` and the start of the next doc-string.

Another nit-pick, when I suggested the doc-strings, I typoed and capitalised the word `Resources` rather than `event`, so if you don't mind, could you do lower case `resources`, and upper case `Event` (in all places in this PR).

Reply:
done and done